.github/workflows/bumpVersions.yml

```diff
@@ -27,6 +24,8 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v3
+        with:
```

Comment:
Please add a comment to explain why we use that token rather than the default one